repos/system_upgrade/el8toel9/actors/checkvdo/libraries/checkvdo.py

```diff
@@ -0,0 +1,180 @@
+import os
+
+from leapp import exceptions
+from leapp import models
+from leapp import reporting
+from leapp.libraries import stdlib
+
+def _run_cmd(command, checked = True):
+    """ Run the specified command returning the result. """
+    try:
+        result = stdlib.run(command, checked = checked)
+    except stdlib.CalledProcessError:
+        raise exceptions.StopActorExecutionError('Failure to execute "{0}"'.format(' '.join(command)))
```

Comment:
Forcing the upgrade to fail because of this without any useful additional information to the user is too harsh.

Reply:
@vinzenz 
Removed trapping of CalledProcessError in commit 2de80cf based on the premise that leapp 'does the right thing' with this exception.